__docs__/wonder-blocks-search-field/search-field.stories.tsx

```diff
@@ -16,7 +16,7 @@ import packageConfig from "../../packages/wonder-blocks-search-field/package.jso
 
 export default {
     component: SearchField,
-    title: "Search Field/SearchField",
+    title: "SearchField",
```

Comment:
**question:** oh so that means that due to this key missing, Chromatic wasn't snapshotting these files?

Reply:
No, I don't think that was the case. I just updated it because I noticed that there was only one component in the search field package, so I updated it to match the other components with the same structure. I noticed it when I was copying the link for it.

Reply:
I see the new snapshots for SearchField and Toolbar though 🤔